delegation/src/Pos/Delegation/Types.hs

```diff
@@ -45,11 +44,6 @@ instance Buildable DlgUndo where
 -- | Map from issuer public keys to related heavy certs.
 type DlgMemPool = HashMap PublicKey ProxySKHeavy
 
--- | Heavyweight PSK with real leader public key (because heavyweight
--- psks have redelegation feature, so pskIssuerPk hPsk /= leader in
--- general case). This is used to create a block header only.
-type ProxySKBlockInfo = Maybe (ProxySKHeavy, PublicKey)
```

Comment:
Moved to https://github.com/input-output-hk/cardano-sl/pull/3089/files#diff-fa0ea3ee19669fdbd25e81f59eeae11dR39